appeals/web/src/server/appeals/appeal-details/appeal-details.mapper.js

```diff
@@ -217,6 +255,7 @@ export async function appealDetailsPage(
 		...notificationBanners,
 		...statusTagsComponentGroup,
 		caseSummary,
+		caseNotesComponent,
```

Comment:
Ideally we should add unit tests to check this is rendering as expected (probably for the POST handler too)